tests/back-end/integration/resources/code-with-us/status.test.ts

```diff
@@ -0,0 +1,96 @@
+import { PG_CONFIG } from "back-end/config";
+import { connectToDatabase } from "back-end/index";
+import {
+  awardCWUProposal,
+  closeCWUOpportunities,
+  readOneCWUOpportunity,
+  readOneCWUProposal,
+  updateCWUOpportunityVersion
+} from "back-end/lib/db";
+import { CWUOpportunityStatus } from "shared/lib/resources/opportunity/code-with-us";
+import { CWUProposalStatus } from "shared/lib/resources/proposal/code-with-us";
+import { getValidValue } from "shared/lib/validation";
+import { clearTestDatabase } from "../../helpers";
+import {
+  setupCWUScenario1,
+  testCreateCWUOpportunityParams
+} from "../../helpers/cwu-scenario-1";
+
+describe("CWU Proposal Resource", () => {
+  afterEach(async () => {
+    await clearTestDatabase();
+  });
+
+  it("should properly update opportunity and proposal statuses when an opportunity is closed and awarded", async () => {
+    const connection = await connectToDatabase(PG_CONFIG);
+
+    // Setup scenario (create an opportunity with two submitted vendor proposals)
+    const { cwuOpportunity, cwuProposal1, cwuProposal2, testAdminSession } =
+      await setupCWUScenario1();
```

Comment:
Are the scenarios going to be used anywhere else? My only qualm is that I need to hop back and forth between files to gather context about the scenario.

Reply:
I made them thinking they could be useful for testing other functionality. It was sort of modelled after the postman tests, which had to be run in sequence to set things up (like creating a RFQ for an org, inviting team members, etc.). I'm totally open to re-arranging these scenarios or putting them inline though if you find it would be more useful that way.